Fixes #173

frostyfan109 opened the pull request:
Adds a `display: none` rule to the ::after pseudo-element which was causing issues, since it didn't seem to be performing any function in the first place.

mbwatson reviewed (changes requested):
i think that instead of using display: none for that pseudoelement, we should just remove it from the code. along with the same reference on lines 54-56. what do you think about that?

frostyfan109 commented:
Updated the fix addressing #173
- Change pseudo-element's `linear-gradient` so that it works properly with Safari (only has partial support). 
- Change pseudo-element's `position:fixed` relative to `.result-card.ant-card` to be explicitly  `position:absolute` relative to `.result-card.ant-card` for browser compatibility.

Fix padding/text positioning issue with `.ant-card-body` where text appeared to have extra top padding due to an inline display rule applied by `.ant-space`.

> Before
<img width="515" alt="image" src="https://user-images.githubusercontent.com/31631417/152234634-eaa20191-53ea-4d25-a96f-bff01aaca96b.png">

> After
<img width="515" alt="image" src="https://user-images.githubusercontent.com/31631417/152234713-8af87c04-ecbd-4eef-992b-c03fb1e57a13.png">

mbwatson reviewed (approved):
👍
